server/service/osquery_utils/queries.go

```diff
@@ -910,7 +910,7 @@ var softwareChrome = DetailQuery{
   'Browser plugin (Chrome)' AS type,
   'chrome_extensions' AS source,
   '' AS vendor,
-  path AS installed_path
+  '' AS installed_path
```

Comment:
This is the code change for number 1. 

To let the user know the chrome extension software will not have an installed path, doc changes are found in `Understanding-host-vitals.md`.